projects/app/src/__tests__/data/rizzle.test.ts

```
@@ -932,9 +934,161 @@ void test(`replicache() mutator tx`, async () => {
+      checkPointsReached++;
+      assert.deepEqual(options, {
+        prefix: `text/`,
+        start: {
+          exclusive: true,
+          key: `text/1:2.`,
+        },
+      });
+      return {
+        async *entries() {
+          return;
+        },
```

Biome (1.9.4)

[error] This generator function doesn't contain yield.



(lint/correctness/useYield)